tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -73,6 +73,24 @@ $(PRJDIR)$(3)/$(1)/third_party/%: tensorflow/lite/micro/tools/make/downloads/% t
 	@mkdir -p $$(dir $$@)
 	@cp $$< $$@
 
+ifeq ($(TARGET_ARCH), arc)
```

Comment:
Can you break out this into a separate generate_synposys_project function, like we do for generate_esp_project, rather than modifying generate_project conditionally?

Reply:
done